src/components/EditHeading/index.js

```diff
@@ -64,42 +62,37 @@ export default class EditHeading extends PureComponent {
 
     const title = permissions.view ?
       (
-        <a style={{ cursor: 'pointer' }} onClick={exitView}>
-          {getModelMessage(i18n, 'model.name')}
-        </a>
+        <ConfirmUnsavedChanges
+          handleConfirm={exitView}
```

Comment:
remove "handleConfirm" property and move handler into a appropriate child property - thus allowing plural children.